pkg/task/restore.go

```diff
@@ -291,15 +291,18 @@ func RunRestore(c context.Context, g glue.Glue, cmdName string, cfg *RestoreConf
 	// Restore has finished.
 	updateCh.Close()
 
-	// Checksum
-	updateCh = g.StartProgress(
-		ctx, "Checksum", int64(len(newTables)), !cfg.LogProgress)
-	err = client.ValidateChecksum(
-		ctx, mgr.GetTiKV().GetClient(), tables, newTables, updateCh)
-	if err != nil {
-		return err
+	// Don't do checksum if user specified.
+	if cfg.Checksum {
```

Comment:
sometimes we cannot do`ValidateChecksum` when there is no checksum data store in `backupmeta`, so I think maybe we can add a check requirement, whether we can do `ValidateChecksum` even user specified

Reply:
That would be good. Let me try it.